dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -1136,6 +1136,17 @@ def assert_caching_enabled(cache_control_header, max_age, proxy_max_age)
     assert_select 'button', I18n.t('teacher.panel.example')
   end
 
+  test "can view CSF teacher markdown as non-authorized teacher" do
+    refute can_view_teacher_markdown?
+
+    stubs(:current_user).returns(@teacher)
+    @script.stubs(:k5_course?).returns(true)
+    assert can_view_teacher_markdown?
+
+    stubs(:current_user).returns(@student)
+    refute can_view_teacher_markdown?
+  end
```

Comment:
I'm not sure how much value there is in a `refute can_view_teacher_markdown?` standalone test.  The goal here is to verify initial assumptions for the rest of the test.